16.md

```diff
@@ -11,7 +11,7 @@ Relays may decide to allow replaceable and/or ephemeral events.
 Replaceable Events
 ------------------
 A *replaceable event* is defined as an event with a kind `10000 <= n < 20000`.
-Upon a replaceable event with a newer timestamp than the currently known latest replaceable event with the same kind, the old event SHOULD be discarded and replaced with the newer event.
+Upon a replaceable event with a newer timestamp than the currently known latest replaceable event with the same kind being received, the old event SHOULD be discarded and replaced with the newer event.
```

Comment:
link to nip-33 somewhere around here? if someone's implementing nip-16, they'll probably want to know about nip-33, too.